pyrmq/publisher.py

```diff
@@ -130,9 +130,10 @@ def connect(self, retry_count=1) -> (BlockingConnection, BlockingChannel):
             return connection, channel
 
         except CONNECTION_ERRORS as error:
-            if not (retry_count % self.connection_attempts):
-                self.__send_reconnection_error_message(retry_count, error)
-                return
+            self.__send_reconnection_error_message(retry_count, error)
```

Comment:
This will send message on every retry. We don't want that.

Reply:
But `self.__create_connection` has already tried three times before this point.

Reply:
Okay. Then retry_count here is not accurate. It will say it retried up to 1 times, 2 times, etc.